website/static/js/osfHelpers.js

```diff
@@ -774,6 +774,12 @@ var confirmDangerousAction = function (options) {
     bootbox.dialog(bootboxOptions);
 };
 
+/** A future-proof getter for the current user
+**/
+var currentUser = function(){
+    return window.contextVars.currentUser;
```

Comment:
Is there ever a case when currentUser wouldn't be in contextVars and if so do we want to return something other than null?